IOS/SwiftSeedProject/SwiftSeedProject/Domain/Source.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  Source.swift
+//  SwiftSeedProject
+//
+//  Created by Brian Sztamfater on 12/4/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class Source {
+    
+    var id: String!
+    var name: String?
+    var description: String?
+    var url: URL?
+    var language: String?
```

Comment:
Can you define this as `var language : Languate`  ? (since you already defined the enum)

Also, check if these fields can be defined as `let` too. Such values shouldn't change once instantiated.